qkernel/src/kernel_def.rs

```diff
@@ -50,6 +50,7 @@ use super::qlib::*;
 use super::syscalls::sys_file::*;
 use super::Kernel::HostSpace;
 
+
```

Comment:
This extra line seems unnecessary, shall we execute a cargo fmt for all codes after all commits rebased?

Reply:
Sure, but I'll limit the format only to the added code.